platform/darwin/scripts/generate-style-code.js

```diff
@@ -159,7 +159,7 @@ global.objCTestValue = function (property, layerType, arraysAsStructs, indent) {
                 `@"'${_.startCase(propertyName)}'"` :
                 `@"${_.startCase(propertyName)}"`;
         case 'string':
-        case 'image': // TODO: replace once we implement image expressions
+        case 'resolvedImage': // TODO: replace once we implement image expressions
```

Comment:
We'll want to make these changes in the https://github.com/mapbox/mapbox-gl-native-ios repo. 

Similarly the Android ones in https://github.com/mapbox/mapbox-gl-native-android. Correct @tobrun?

Reply:
yes ticket in https://github.com/mapbox/mapbox-gl-native-android/issues/33